core/services/vrf/v2/coordinator_v2x_interface.go

```diff
@@ -36,9 +36,10 @@ type CoordinatorV2_X interface {
 	GetConfig(opts *bind.CallOpts) (Config, error)
 	ParseLog(log types.Log) (generated.AbigenLog, error)
 	OracleWithdraw(opts *bind.TransactOpts, recipient common.Address, amount *big.Int) (*types.Transaction, error)
+	Withdraw(opts *bind.TransactOpts, recipient common.Address) (*types.Transaction, error)
```

Comment:
Do we need a second method here for withdrawing native?

Reply:
Yeah looks like `WithdrawNative()` could use a test 👍

Reply:
In terms of test coverage, withdrawNative is tested in integration tests which uses a different interface from this one.

Reply:
So this interface is not tested or is just the `Withdraw()` method not used/tested? And why do integration tests use a different interface?

Reply:
The interface is used and tested but seems that a few methods (`withdraw()` and `withdrawNative()`) are unused. 

The integration tests use different interface but I'm not sure if we have a good reason for it. I think we have room for some refactoring in integration tests to either create a shared interface for v2 and v2plus or use the existing one above.